lapce-ui/src/CmdLineArgs.rs

```diff
@@ -0,0 +1,166 @@
+extern crate getopts;
+use getopts::Options;
+use std::env;
+
+pub fn version(executable: &str){
+   	format!("Usage: {} [options]", executable);
+    println!("Lightning-fast and Powerful Code Editor: Check details at https://lapce.dev");
+}
+
+pub fn help(executable: &str, opts: Options) {
+        let error = format!("Usage: {} [options]", executable);
+        println!("{}", opts.usage(&error));
+        println!("If you are having trouble with Lapce setup, please visit https://docs.lapce.dev");
+        println!("If you wish to contribute, please go to https://github.com/lapce/lapce.git");
+}
+
+
+pub fn start_cmd_line() -> bool {
+    let args: Vec<String> = env::args().collect();//get arguments
+    let exec = args[0].clone();//executable
+    let mut opts = Options::new();
+    opts.optflag("v", "version", "check Lapce version");
+    opts.optflag("h", "help", "help");
+    //additional new options go here
+    let matches = match opts.parse(&args[1..]) {
+        Ok(m) => { m }//accepts from Ok()
+        Err(f) => {panic!("{}. Use --help to list recognized commands", f.to_string()); return false;}//rejects from Err()
+    };
+    if matches.opt_present("h") {
+        help(&exec, opts);
+        return true;
+    }
+    if matches.opt_present("v"){
+        version(&exec);
+        return true;
+    }
+    return true;
+}
+
+
+#[cfg(test)]
+mod tests {
+    // Note this useful idiom: importing names from outer (for mod tests) scope.
+    use super::*;
+
+    #[test]
+    fn test_stub(){}
+    fn test_general() {
```

Comment:
The test framework will only run functions marked with the `#[test]` attribute, and here only the first (empty) function is marked as such.

Anyway, what's the purpose of this function? It doesn't seem to test anything useful.

Reply:
Added your changed! 

Yreah, you are right, it does not do anything, I was just checking to see if the test cases were actually working, forgot to delete the stub. I deleted it in the new commit!

Reply:
Thanks! I would like to point out that still only one function is a test, the others will not run.